### ISSUE TITLE
custom yaml tags are ignored

### ISSUE THREAD

**Issue by jia2** [GoogleCloudPlatform/cloud-code-vscode#122]
**Environment**:
Extension version: 0.0.9
VSCode version: 1.36.1
OS: Windows 10, build 1809

**Description**: 
After installing this plugin (version 0.0.9), all custom yaml tags are ignored by yaml parser. It seems there is a confict with yaml parser (https://github.com/redhat-developer/yaml-language-server)

**Repro step**:
define custom yaml tag in user's settings.json and install this plugin, creat a yaml file and use defined custom tag, then yaml parser will report "unkown tag".

**Comment by redborian** [comment]
Thanks for using our extension. Please copy the contents from your `yaml.customTags` setting into `cloudcode.yaml.customTags`. This should make our language server aware of the custom tags.

**Comment by jia2** [comment]
Thanks for your quick reply.
I copied `yaml.customTags` setting into `cloudcode.yaml.customTags`, and observed that the language server in cloud code extension interpreted custom tags different as YAML Language Server of redhat. I didn't know yaml spec well and can't tell, which parses yaml file with custom tags correctly according to yaml specification.

**Comment by redborian** [comment]
Do you mind sharing the contents of `customTags`? We will investigate why that is the case and get back to you.

**Comment by jia2** [comment]
If I put `"!Sub scalar"` before `"!Sub sequence"` in `"cloudcode.yaml.customTags"`, 
```
"cloudcode.yaml.customTags": [
        "!Sub scalar",
        "!Sub sequence",
        ...
    ]
```


I got 133 parsing errors, but when I moved  `"!Sub scalar"` after `"!Sub sequence"`, 

```
"cloudcode.yaml.customTags": [
        "!Sub sequence",
        "!Sub scalar",
        ...
    ]
```
then errors are reduced to 12.

And I have lines blow in my yaml file:

```yaml
createMonitoring:
    commands:
        createEc2Monitoring:
            command: !Sub
                - "bash /tmp/monika-monitoring/create_ec2_alarms.sh -A ${AccountId} -V ${Vpc} -a sresb -c mmp -e ${Env}"
                - { AccountId: !Ref "AWS::AccountId", Vpc: !ImportValue VPC1-VPC-ID }
```

This snippet comes from one of my aws cloudformation template in yaml format.

And for these lines I got below errors, no matter what kind of kombinations of custom tags I used.

unacceptable node kind for !<!Sub> tag; it should be "scalar", not "sequence"
incomplete explicit mapping pair; a key node is missed

With redhat yaml languagae server there is no parse error with the same custom tags for the same yaml file. I'm not quit sure, that my yaml file has correct syntax, but redhat yaml language server believe it does.

**Comment by redborian** [comment]
Just to give you an update on this issue, we are on working on merging in changes that fixes this issue. I will update this thread once it is merged in.

**Comment by redborian** [comment]
This should be fixed in latest release (v0.0.13)
